Package.md

```diff
@@ -0,0 +1,241 @@
+Azure EventHub Connector
+
+# Module Overview
+
+Azure Event Hubs is a highly scalable data ingress service that ingests millions of events per second so that you can process and analyze the massive amounts of data produced by your connected devices and applications. Once data is collected into an Event Hub, it can be transformed and stored using any real-time analytics provider or batching/storage adapters.
+
+The connector will only be focusing on sending events to the event hub. The event hub connector will invoke the REST APIs exposed via the Azure Event Hub. https://docs.microsoft.com/en-us/rest/api/eventhub/. 
+
+The REST APIs fall into the following categories:
+
+- Azure Resource Manager: 
+  APIs that perform resource manager operations, and have /providers/Microsoft.EventHub/ as part of the request URI.
+
+- Event Hubs service: 
+  APIs that enable operations directly on the Event Hubs service, and have <namespaceName>.servicebus.windows.net/ in the request URI. The Event Hubs service API is focused on this implementation. 
+
+## Compatibility
+|                     |    Version          |
+|:-------------------:|:-------------------:|
+| Ballerina Language  | swan-lake-preview8  |
+
+
+## Samples:
+
+1. Sending an event.
+
+```ballerina
+import ballerinax/azure_eventhub as eventhub;
+
+public function main() {
+   eventhub:ClientEndpointConfiguration config = {
+       sasKeyName: "<sas_key_name>",
+       sasKey: "<sas_key>",
+       resourceUri: "<resource_uri>"
+   };
+   eventhub:Client c = new (config);
+   var b = c->send("myhub", "eventData");
+}
+```
+
+2. Sending an event with broker properties and user properties.
+
+```ballerina
+import ballerinax/azure_eventhub as eventhub;
+
+public function main() {
+   eventhub:ClientEndpointConfiguration config = {
+       sasKeyName: "<sas_key_name>",
+       sasKey: "<sas_key>",
+       resourceUri: "<resource_uri>"
+   };
+   eventhub:Client c = new (config);
```

Comment:
Can't we give more meaning than b, c in this place?

Reply:
Resolved Thank you